interpreter/cling/lib/Interpreter/ForwardDeclPrinter.cpp

```diff
@@ -1199,17 +1228,28 @@ namespace cling {
         Expr* TAExpr = TA.getAsExpr();
         if (CastExpr* CastExpr = dyn_cast<clang::CastExpr>(TAExpr))
           TAExpr = CastExpr->getSubExpr();
-        if (DeclRefExpr* DRE = dyn_cast<DeclRefExpr>(TAExpr)) {
+        else if (DeclRefExpr* DRE = dyn_cast<DeclRefExpr>(TAExpr)) {
```

Comment:
With that `else statement`, isn't the previous `line 1230: TAExpr = CastExpr->getSubExpr();` unnecessary/unused?

Reply:
Good catch, thanks!